test/mocha/mock.plugin.js

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright (c) 2017-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+module.exports = {
+  type: 'service',
```

Comment:
Why is the type `service` here? Shouldn't it be something like `storagePlugin`? I think the `type` is used to identify the expected API. `service` is being used for ledger-agent plugins which use a `router` in the API -- these use `index`, `storage,` etc in the API. Perhaps `service` should be `ledgerAgentPlugin` or something? How should we be naming these?

cc: @msporny, @davidlehn

Reply:
-1 to `service` ... I don't know what the means by looking at it. +1 to `storagePlugin` or something of that nature...